addons/event_hr_skills/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+{
+    'name': 'Events HR Skills',
+    'category': 'Marketing/Events',
+    'summary': "Manage Employees' Events",
+    'description': """
+Link Employees to the events they attend, and display them on their resume.
+""",
+    'depends': ['hr_skills', 'event'],
+    'data': [
+        'views/event_category_tag_views.xml',
+        'views/hr_resume_line_views.xml',
+        'data/hr_resume_data.xml',
+    ],
+    'demo': [
+        'data/event_demo.xml'
```

Comment:
```suggestion
        'data/event_hr_skills_demo.xml'
```
should be event_hr_skills_demo.xml :)